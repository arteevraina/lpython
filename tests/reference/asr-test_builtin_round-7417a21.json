{
    "basename": "asr-test_builtin_round-7417a21",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/test_builtin_round.py",
    "infile_hash": "a4ff4032a45ce084eb524069046afb6ec44f1b24a667c84c7605f2c7",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_builtin_round-7417a21.stderr",
    "stderr_hash": "a58af3b9f4312b1036b728dc70928fe9c05f87d1b96d92b5c3da881f",
    "returncode": 2
}