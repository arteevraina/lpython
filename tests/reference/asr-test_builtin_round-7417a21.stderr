warning: The module 'lpython_builtin' located in $DIR/src/bin/../runtime/lpython_builtin.py cannot be loaded
 --> tests/../integration_tests/test_builtin_round.py:6:12
  |
6 |     assert round(f) == 6
  |            ^^^^^^^^ imported here

semantic error: The symbol '_lpython_floordiv' not found in the module 'lpython_builtin'
   --> $DIR/src/bin/../runtime/lpython_builtin.py:209:15
    |
209 |     if (n_ - (n_ // 2)*2) == 0:
    |               ^^^^^^^ 
